lib/libmalloc_simple/heap.c

```diff
@@ -47,7 +47,7 @@ static char *rcsid = "$FreeBSD$";
 #include <sys/param.h>
 #include <sys/types.h>
 #ifdef CAPREVOKE
-#include <sys/caprevoke.h>
+#include <cheri/revoke.h>
```

Comment:
This commit may be harder to squash, and perhaps you want the history, but otherwise I would suggest squashing this into the previous two commits so we just have the current revoker API used.